homeassistant/components/surepetcare/services.yaml

```diff
@@ -20,3 +20,24 @@ set_lock_state:
             - 'locked_in'
             - 'locked_out'
             - 'unlocked'
+
+set_pet_location:
+  name: Set pet location
+  description: Set pet location
+  fields:
+    pet_name:
+      description: Name of pet
+      example: My_cat
+      required: true
+      selector:
+        text:
+    location:
+      description: Pet location (Inside or Outside)
+      example: inside
+      required: true
+      selector:
+        select:
+          options:
+            - 'Inside'
```

Comment:
Is it really necessary to specify the options here, when they are included in the service schema?

Reply:
you mean how services are defined is 'broken by design'? :>

Reply:
yup it is, at least I do not know any other way, I assume it is a left over from some old way of doing things. The real question is why do we need services.yaml at all, we register the services that should be enough.